packages/webview_flutter/lib/platform_interface.dart

```diff
@@ -21,8 +21,7 @@ abstract class WebViewPlatformCallbacksHandler {
   /// Invoked by [WebViewPlatformController] when a navigation request is pending.
   ///
   /// If true is returned the navigation is allowed, otherwise it is blocked.
-  FutureOr<bool> onNavigationRequest(
-      {required String url, required bool isForMainFrame});
+  FutureOr<bool> onNavigationRequest({required String url, required bool isForMainFrame});
```

Comment:
I can revert formatting changes if required, but I just did a `flutter format --line-length=80 .` to ensure my own code was formatted correctly, so I guess some existing code wasn't.